### PR TITLE
Update ISic000482.xml

### PR DESCRIPTION
**Pull request by alessandrarocchetti**

### CODE REVIEW

File: inscriptions/ISic000482.xml [hunk 1]
```diff
@@ -183,20 +188,31 @@
...
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>All'imperatore Caesare Traiano Adriano Augusto, figlio del divo imperatore Traiano Partico, pontefice massimo, nella sua terza tribunicia potesta, console per la terza volta. Luogo concesso per decreto dei decurioni.</p>
             </div>
+            <div type="translation" xml:lang="en" resp="#AR">
+                <p>For the emperor Caesar Trajan Hadrian Augustus, son of the diefied emperor Trajan, victor over the Partian, chief priest, holder of the tribunician powers three times, consul three times. The place was given by decree of the town councillors.</p>
             <div type="commentary">
                 <p><!--commented out pending revision-->
                     <!--note that 7208 is on the same block. Inscribed on a column capital: to Trajan, with titles, and ends I.D.D.D. - but possibly this should be LDDD, given that the bottom edge is broken off, and LDDD makes ready sense, IDDD seems more difficult.--></p>
```
Comment (alessandrarocchetti) [inline inscriptions/ISic000482.xml:218]: I think the emperor is Hadrian not Trajan ... I might be wrong though

File: inscriptions/ISic000482.xml [hunk 2]
```diff
@@ -183,23 +188,34 @@
...
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/><supplied reason="lost"><expan><abbr>imp</abbr><ex>eratori</ex></expan></supplied> <supplied reason="lost"><expan><abbr>caes</abbr><ex>ari</ex></expan></supplied> <supplied reason="lost">divi</supplied> <supplied reason="lost">traiani</supplied>
+                    <lb n="2"/><supplied reason="lost">parthi</supplied>ci <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>elix</ex></expan> <g ref="#interpunct">·</g> <name>Traiano</name> <name>Ha<supplied reason="lost">dri</supplied>
```
Comment (JonPrag) [inline inscriptions/ISic000482.xml:198]: filio, not felix (son of the deified Traianus Parthicus)

File: inscriptions/ISic000482.xml [hunk 3]
```diff
@@ -183,23 +188,34 @@
...
+                    <lb n="3" break="no"/><supplied reason="lost">an</supplied>o</name> <g ref="#interpunct">·</g> <expan><abbr>Aug</abbr><ex>usto</ex></expan> <expan><abbr>pontif</abbr><ex>ice</ex></expan> <g ref="#interpunct">·</g>  <expan><abbr>max</abbr><ex>imo</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>trib</abbr><ex>unicia</ex></expan> <g ref="#interpunct">·</g>
```
Comment (JonPrag) [inline inscriptions/ISic000482.xml:199]: pontifici (dative, not ablative)

File: inscriptions/ISic000482.xml [hunk 4]
```diff
@@ -183,23 +188,34 @@
...
+                    <lb n="4"/><expan><abbr>pot</abbr><ex>esta</ex></expan> <g ref="#interpunct">·</g> <num value="3">III</num> <g ref="#interpunct">·</g> <expan><abbr>co</abbr><ex>n</ex>s<ex>ule</ex></expan> <num value="3">III</num>
```
Comment (JonPrag) [inline inscriptions/ISic000482.xml:200]: consuli, not consule

File: inscriptions/ISic000482.xml [hunk 5]
```diff
@@ -183,23 +188,34 @@
...
+                    <lb n="5"/><expan><abbr><choice><reg>l</reg><orig>i</orig></choice></abbr><ex>ocus</ex></expan> <expan><abbr>d</abbr><ex>atus</ex></expan> <expan><abbr>d</abbr><ex>ecreto</ex></expan> <expan><abbr>d</abbr><ex>ecurionum</ex></expan>
```
Comment (JonPrag) [inline inscriptions/ISic000482.xml:201]: unclear for the L, not reg versus orig, so:
`<expan><abbr><unclear>l</unclear></abbr><ex>ocus</ex></expan>`

File: inscriptions/ISic000482.xml [hunk 6]
```diff
@@ -183,23 +188,34 @@
...
+                    </app>
+                </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>All'imperatore Caesare Traiano Adriano Augusto, figlio del divo imperatore Traiano Partico, pontefice massimo, nella sua terza tribunicia potesta, console per la terza volta. Luogo concesso per decreto dei decurioni.</p>
             </div>
+            <div type="translation" xml:lang="en" resp="#AR">
+                <p>For the emperor Caesar Trajan Hadrian Augustus, son of the diefied emperor Trajan, victor over the Partian, chief priest, holder of the tribunician powers three times, consul three times. The place was given by decree of the town councillors.</p>
```
Comment (JonPrag) [inline inscriptions/ISic000482.xml:215]: <p>For the Emperor Caesar Trajan Hadrian Augustus, son of the deified Trajan Parthicus, chief priest, holder of the tribunician power for the third time, consul for the third time. The place was given by decree of the town councillors.</p>

**Review by JonPrag — CHANGES_REQUESTED** [review]
Thanks Alexandra - sorry it's taken me so long- various minor things to tweak.

**Review by JonPrag — APPROVED** [review]

**Pull request merged**